jxp/My-List-Util/t/shuffle.t

```diff
@@ -0,0 +1,18 @@
+#!perl -T
+use 5.006;
+use strict;
+use warnings;
+use Test::More tests => 8;
+use My::List::Util;
+
+  my @list=(1,2,3,4);
+
+  my @new_list=My::List::Util::shuffle(@list);
+  
+  foreach my $value ( @list ){
+    ok(grep{$_ eq $value} @new_list, "$value is in new list");
+  }
+  
+  foreach my $value ( @new_list ){
+    ok(grep{$_ eq $value} @list, "$value is in old list");
+  }
```

Comment:
This does check that no values are lost or new values added, though it would be possible to pass this test by duplicating values.